qubes-gui-client/src/buffer.rs

```diff
@@ -117,7 +117,25 @@ fn u32_to_usize(i: u32) -> usize {
     i as usize
 }
 
-impl<T: VchanMock> RawMessageStream<T> {
+#[derive(Debug)]
+pub(crate) struct ClaimableBuffer<'a, T: VchanMock> {
+    inner: &'a mut RawMessageStream<T>,
+    hdr: Header,
+}
```

Comment:
We don't need a reference to the entire `RawMessageStream`, a `&mut Vec<u8>` is enough. This removes the need for the `T` type parameter and the `qubes_gui_client::Buffer` wrapper around `ClaimableBuffer`.